azure_costs_exporter/views.py

```diff
@@ -11,14 +12,19 @@ def health():
 @bp.route("/metrics", methods=['GET'])
```

Comment:
Prometheus also comes with a built-in web service, see here: https://github.com/prometheus/client_python#http